quickfix/AttributeRenameDefault.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright (c) 2014-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict'
+
+	CKEDITOR.plugins.a11ychecker.quickFixes.get( {
+		name: 'AttributeRename',
+		callback: function( QuickFix ) {
+			function AttributeRenameDefault(issue) {
+				QuickFix.call(this, issue);
+			}
+
+			AttributeRenameDefault.prototype = new QuickFix();
+
+			AttributeRenameDefault.prototype.constructor = AttributeRenameDefault;
+
+			CKEDITOR.tools.extend( AttributeRenameDefault.prototype, {
```

Comment:
No need to use `CKEDITOR.tools.extend` on prototype in this case, so far there was no need for it so we're sticking to vanilla js.

Actually I think that instead of overriding `display` method, you could do a minor refactoring in `AttributeRename.display` - you can just extract the value obtaining into a method. Then override this method in `AttributeRenameDefault` type.